extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -186,12 +190,15 @@ export class AmpAnalytics extends AMP.BaseElement {
 
           if (trigger['selector']) {
             // Expand the selector using variable expansion.
-            trigger['selector'] = this.expandTemplate_(trigger['selector'],
-                trigger, /* arg*/ undefined, /* arg */ undefined,
-                /* arg*/ false);
-            this.instrumentation_.addListener(
-                trigger, this.handleEvent_.bind(this, trigger), this.element);
-
+            return this.variableService_.expandTemplate(
+                trigger['selector'], trigger, this.config_,
+                /* opt_event */ undefined, /* opt_iterations */ undefined,
```

Comment:
`opt_event` is an object,  `opt_iterations` is a number. `opt_no_encode` is the only boolean :). Making the change :)